### PR TITLE
HTML-CSS-Backgrounds PR

### PR DESCRIPTION
**Pull request by Gabkings**
# Apple's Home Page Clone

> This is a frontend project with the main aim of learning CSS background properties by cloning Apple Home Page.
###  Navigation bar

- It has a search input
- Links

###  Header
- Navbar 
- It has a sample paragraphs
-Background Image

### Phones Sections
 - Phone images with texts

### Screenshorts

review

bsg2

## Built With

- HTML5,
- CSS3

## Getting Started

**This is an example of how you may give instructions on setting up your project locally.**
**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the current project doesn't require testing.**

To get a local copy up and running follow these simple example steps.

### Prerequisites
- HTML5 semantic tags
- CSS3 
- Git Workflow

### Setup
- Clone the repo: ```https://github.com/Gabkings/css-bqackgrounds.git ```

### Install
- Open index.htnl with the browser of your choice

## Authors
👤 **Author1**

- Github: [@githubhandle](https://github.com/Gabkings)

- Linkedin: [linkedin](https://www.linkedin.com/in/gabriel-gitonga-b5a611183/)

## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](issues/).

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Hat tip to anyone who's code was used
- Inspiration
- etc

## 📝 License

This project is [MIT](lic.url) licensed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            Watch the "Your Verse" film and explore the stories
+            <i class="fa fa-play"></i>
+          </p>
+        </div>
+      </div>
+      <div class="phones">
+        <div><img src="img/image4.jpg" alt="image" /></div>
+        <div><img src="img/image3.jpg" alt="image" /></div>
+        <div><img src="img/image2.jpg" alt="image" /></div>
+        <div><img src="img/image1.jpg" alt="image" /></div>
```
Comment (addod19) [inline index.html:49]: Kindly set each div with a class and give them the background images and add a link to each div. The focus for this project is using background and gradients

Comment (addod19) [reply index.html:49]: Let us implement the DRY rule so we avoid repeating code in our CSS file, we should also avoid using ids and tag names for styling

**Comment by uryelah** [comment]
Hello **Gitonga Gitonga**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**Good job with the images and gradients!🌟**

There are still some issues to fix before going to the next project, but you got this!💪 

---------     

## Required Changes 🏗 👷‍♀ 

- [ ] Avoid using id selectors for styling, [read this article on why it's better to use classes](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4), make sure to change the id selectors to class selectors.

- [ ] Remember to follow [commit writing best practices ](https://chris.beams.io/posts/git-commit/)from now on:
  - Write your commit messages using the imperative tense
  - Capitalize your commits

- [ ] Change the font-family of both the `.header` div and the footer to match more closely the font of the [original page.](https://web.archive.org/web/20140301004610/http://www.apple.com/)

- [ ] Make sure the navbar is located vertically from the top of the page in a similar distance that the original one is.

- [ ] Remove the blue color from the nav gradient.

- [ ] Change the color and size of the header text to it matches the original.

- [ ] [OPTIONAL] Add the hover interaction the original navbar has.

Compare here your navbar and the original one:
Screenshot_2020-04-29 CSS Backgrounds and Gradients
Screenshot_2020-04-29 Apple

----

Compare here your header text and the original one:
Screenshot_2020-04-29 CSS Backgrounds and Gradients(1)
Screenshot_2020-04-29 Apple(1)


----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by uryelah** [comment]
Hey Gitonga, good job with most of the changes but some things scaped a bit.

---------     

## Required Changes 🏗 👷‍♀ 

- [ ] Please change the weight and color of the header text so it's thin and white like the original one.

- [ ] For some reason I can't find your readme file filled on either branch. Please add the readme content you used on your PR message on the actual readme file.

----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by trekab** [comment]
Hey Gitonga, great job :+1:

##Required Changes:
 It appears that your readme file lacks the necessary content. Please add the readme content you used on your PR message on the actual readme file.

__Pleas fix the REQUIRED issues and send another code review request.__
Happy coding!
Code review by _Treasure Kabareebe_

**Comment by trekab** [comment]
# Status Approved :+1: :1st_place_medal:
Great job with the changes, @Gabkings.

**Pull request merged**